tmva/tmva/test/DNN/CNN/TestMethodDL.h

```diff
@@ -49,14 +49,12 @@ void testMethodDL_CNN(TString architectureStr)
    // Load the input data
 
    TString fname = "imagesData.root";
-   TString fopt = "CACHEREAD";
```

Comment:
Can you mention in the commit log that you removed the unnecessary `CACHEREAD` for the local file?

Reply:
Commit log message updated.